{"created": 1652975425.9696608, "duration": 0.12002110481262207, "exitcode": 1, "root": "D:\\MyProjects\\WebTesterBot", "environment": {"Python": "3.10.2", "Platform": "Windows-10-10.0.22000-SP0", "Packages": {"pytest": "7.1.2", "py": "1.11.0", "pluggy": "1.0.0"}, "Plugins": {"html": "3.1.1", "json-report": "1.5.0", "metadata": "2.0.1"}}, "summary": {"passed": 3, "failed": 1, "total": 4, "collected": 4}, "collectors": [{"nodeid": "", "outcome": "passed", "result": [{"nodeid": "pytest/ch1/test_one.py", "type": "Module"}, {"nodeid": "pytest/ch1/test_three.py", "type": "Module"}, {"nodeid": "pytest/ch1/test_two.py", "type": "Module"}]}, {"nodeid": "pytest/ch1/test_one.py", "outcome": "passed", "result": [{"nodeid": "pytest/ch1/test_one.py::test_passing", "type": "Function", "lineno": 0}]}, {"nodeid": "pytest/ch1/test_three.py", "outcome": "passed", "result": [{"nodeid": "pytest/ch1/test_three.py::test_defaults", "type": "Function", "lineno": 6}, {"nodeid": "pytest/ch1/test_three.py::test_member_access", "type": "Function", "lineno": 12}]}, {"nodeid": "pytest/ch1/test_two.py", "outcome": "passed", "result": [{"nodeid": "pytest/ch1/test_two.py::test_failing", "type": "Function", "lineno": 0}]}], "tests": [{"nodeid": "pytest/ch1/test_one.py::test_passing", "lineno": 0, "outcome": "passed", "keywords": ["test_passing", "pytest/ch1/test_one.py", "WebTesterBot"], "setup": {"duration": 0.0003088000012212433, "outcome": "passed"}, "call": {"duration": 0.00021220000053290278, "outcome": "passed"}, "teardown": {"duration": 0.00019660000180010684, "outcome": "passed"}}, {"nodeid": "pytest/ch1/test_three.py::test_defaults", "lineno": 6, "outcome": "passed", "keywords": ["test_defaults", "pytest/ch1/test_three.py", "WebTesterBot"], "setup": {"duration": 0.00019470000188448466, "outcome": "passed"}, "call": {"duration": 0.00018790000103763305, "outcome": "passed"}, "teardown": {"duration": 0.0001765000015439, "outcome": "passed"}}, {"nodeid": "pytest/ch1/test_three.py::test_member_access", "lineno": 12, "outcome": "passed", "keywords": ["test_member_access", "pytest/ch1/test_three.py", "WebTesterBot"], "setup": {"duration": 0.00019070000053034164, "outcome": "passed"}, "call": {"duration": 0.0001874999979918357, "outcome": "passed"}, "teardown": {"duration": 0.00017929999739862978, "outcome": "passed"}}, {"nodeid": "pytest/ch1/test_two.py::test_failing", "lineno": 0, "outcome": "failed", "keywords": ["test_failing", "pytest/ch1/test_two.py", "WebTesterBot"], "setup": {"duration": 0.00018500000078347512, "outcome": "passed"}, "call": {"duration": 0.0003623000011430122, "outcome": "failed", "crash": {"path": "D:\\MyProjects\\WebTesterBot\\pytest\\ch1\\test_two.py", "lineno": 2, "message": "assert (1, 2, 3) == (3, 2, 3)\n  At index 0 diff: 1 != 3\n  Use -v to get more diff"}, "traceback": [{"path": "pytest\\ch1\\test_two.py", "lineno": 2, "message": "AssertionError"}], "longrepr": "def test_failing():\n>       assert (1, 2, 3) == (3, 2, 3)\nE       assert (1, 2, 3) == (3, 2, 3)\nE         At index 0 diff: 1 != 3\nE         Use -v to get more diff\n\npytest\\ch1\\test_two.py:2: AssertionError"}, "teardown": {"duration": 0.00022510000053443946, "outcome": "passed"}}]}